WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -105,6 +105,10 @@ $venue-map-wrapper-height: 400px;
   }
 }
 
+#map-wrapper {
+  height: 600px;
```

Comment:
I do think this is worth doing, but upon further thought, we can always improve this at a later date. Please ignore my comments about the List vs Map buttons as well, but please do change "commit" to something else.